src/components/Table/helpers/renderCell.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react'
+import { isNull, isUndefined, isFunction } from 'lodash'
+
+import chain from 'helpers/generators/chain'
+import renderText from './renderText'
+import { StyledTableCell } from '../styles'
+
+const renderCell = (col, element, { renderCell }) => {
```

Comment:
Strange pattern to have a named Parameter that is the same as the function's name. Is that intended ?